server/lib/auth.js

```diff
@@ -14,7 +14,8 @@ const minutesToSeconds = minutes => moment.duration({ minutes }).asSeconds();
 export const TOKEN_EXPIRATION_LOGIN = minutesToSeconds(75);
 export const TOKEN_EXPIRATION_CONNECTED_ACCOUNT = daysToSeconds(1);
 export const TOKEN_EXPIRATION_SESSION = daysToSeconds(90);
-export const TOKEN_EXPIRATION_PDF = minutesToSeconds(75);
+export const TOKEN_EXPIRATION_PDF = minutesToSeconds(5);
```

Comment:
Also changed the value for PDF service here, it's a single-use token so 5 minutes should be large enough.